changelog.txt

```diff
@@ -1,3 +1,95 @@
+* Game speeds are now more closely aligned with the original game
+* Improvements to handling of win/lose conditions, and the progress report
+* Staff tiredness levels are now taken from the level config file based on
+  difficulty level of the main campaign
+* You can no longer win a level if you still have outstanding loans
+* Emergencies will now be announced once patients actually begin arriving
+  instead of at the start
+* Errors with music playback will now attempt to provide more helpful
+  information in the console
+* Level briefings now show before the in-game tutorial
+* The first patient of a level will now arrive faster after opening the
+  hospital
+* The information dialog box is now more closely aligned with the original game
+* The tip of the day window should no longer be obscured in the main menu
+* The load/save windows now have better labelling
+* Unavailable languages are shown as disabled until you select a unicode font
+* Support is added to auto detect a Theme Hospital install via GOG Galaxy
+
+# Translations
+* Dutch translation has been updated. Thanks @jetenergy
+* Italian translation has been updated. Thanks @SebastianoPistore & @Inkub0
+* Russian translation has been updated. Thanks @Matroftt
+* Spanish translation has been updated. Thanks @ShiroAka
+* French translation has been updated. Thanks @nMustaki & @Sanndow
+* Some unused language strings have been cleaned up
+* Custom campaign and level creators can now optionally add translated strings
+  for campaign description, level briefing, and winning text
+
+# Bug Fixes
+* Swing (double) doors will no longer crash your game if you built rooms that
+  used them while paused
+* Fixed a bug for NVIDIA users who didn’t like graphical corruption when
+  playing fullscreen at non-native resolutions
+* Games should no longer crash irrecoverably because you have a 4k monitor :) 
+* The game will now exit to the main menu cleanly if a problem occurred trying
+  to load a new level or map
+* Staff who have left the hospital can no longer ask for a raise
+* Fixed a bug where some staff may have no initial before surname
+* Patients can no longer litter outdoors
+* Fixed a bug where an unreachable reception desk could cause a crash
+* The Computer and Atom Analyser now make button sounds as originally intended
+* The mark for vaccination action now makes a sound as originally intended
+* Fixes a rare bug where edges of map tiles for parcels could cause unintended
+  behaviour when purchasing plots
+* Active cheats will now persist across saves
+* Audio settings have better safeguards against no audio enabled/no background
+  music
+* Movies will no longer attempt to play audio when global audio is off
+* Config values using brackets (such as a custom music directory) will now work
+  properly
+* Custom campaigns menu now will use a scrollbar for long campaign descriptions
+* Continue game now properly targets files explicitly ending in the .sav format
```

Comment:
```suggestion
* Continue-game now properly targets files explicitly ending in the .sav format
```

Reply:
Might want to leave the dash out of this one to align with the button in the game

Reply:
Your suggestion makes sense to me.

Reply:
```suggestion
* Continue Game now properly targets files explicitly ending in the .sav format
```